spec/requests/vendors_requests_spec.rb

```diff
@@ -28,10 +28,6 @@
           expect(response.body).to include(first_vendor.business_name)
           expect(response.body).not_to include(deactivated_vendor.business_name)
         end
-
-        it "should have a deactivate button for each active vendor" do
-          expect(subject.body.scan("Deactivate").count).to eq(2)
-        end
```

Comment:
Why was this test removed? It should be re-added with a separate test for non-purchases vendors.